tests/integration-tests/tests/schedulers/test_torque.py

```diff
@@ -233,6 +234,9 @@ def _get_job_state(remote_command_executor, job_id):
 
 
 def _assert_scheduler_configuration(remote_command_executor, torque_commands, max_slots, max_queue_size):
+    # Add wait head node np setting retry for 1 mins, daemons retrieve ASG every 3 mins and there is a time gap
+    # between ASG retrieval and head node np setting, 200 seconds sleep is not enough for this test after updating ASG
+    _wait_head_node_np_setting_complete(max_queue_size, max_slots, torque_commands, remote_command_executor)
     compute_nodes_count = torque_commands.compute_nodes_count()
```

Comment:
Can you pass the compute node count to this function? If not, can you pass it to the function that waits for sqswatcher to update np?